src/request.rs

```diff
@@ -356,6 +368,50 @@ impl<B> RequestBuilder<B> {
         self.allow_compression = allow_compression;
         self
     }
+
+    /// Sets if this `Request` will accept invalid TLS certificates.
+    ///
+    /// The default value is `false`.
+    ///
+    /// # Danger
+    /// Use this setting with care. This will accept **any** TLS certificate valid or not.
+    /// If you are using self signed certificates, it is much safer to add their root CA
+    /// to the list of trusted root CAs by your system.
+    #[cfg(feature = "tls")]
+    pub fn accept_invalid_certs(mut self, accept_invalid_certs: bool) -> Self {
```

Comment:
I know it looks somewhat ugly, the as the original reporter suggested, one could name this something like `danger_accept_invalid_certs` so it immediately stands out during code review?

Reply:
I personally find the `danger_` a bit patronizing but it seems to be the standard that's been set in the Rust community.